RecoLocalMuon/GEMRecHit/python/gemRecHits_cfi.py

```diff
@@ -1,15 +1,11 @@
 import FWCore.ParameterSet.Config as cms
 
 gemRecHits = cms.EDProducer("GEMRecHitProducer",
-    recAlgoConfig = cms.PSet(
-
-    ),
+    recAlgoConfig = cms.PSet(),
     recAlgo = cms.string('GEMRecHitStandardAlgo'),
-    gemDigiLabel = cms.InputTag("simMuonGEMDigis"),
+    gemDigiLabel = cms.InputTag("muonGEMDigis"),
```

Comment:
Now that muonME0Digis are produced, shouldn't they be replaced in me0RecHits_cfi.py, as it is done for muonGEMDigis here?

Reply:
yes, eventually. Currently me0RecHits takes simMuonME0PseudoReDigis.
we will change this later once me0 design is set